docs/source/distributed.tensor.parallel.rst

```diff
@@ -57,4 +57,4 @@ Users just need to call the following API explicitly:
 
 
 .. currentmodule:: torch.distributed.tensor.parallel.fsdp
-.. autofunction::  is_available
+.. autofunction::  enable_2d_w_fsdp
```

Comment:
nit about naming: shall we just call `enable_2d_with_fsdp`?

Reply:
Sure, I was just thinking it might be too long? I don't have a strong opinion here.